src/components/Input/WalletInput.test.tsx

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable new-cap */
+import ReactDOM from 'react-dom';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import 'jest-styled-components';
+import 'jest-canvas-mock';
+import { composeStories } from '@storybook/testing-react';
+import * as stories from './Input.stories';
+import color from '../../styles/colors';
+import React from 'react';
+import rgbToHex from '../../utils/rgbToHex';
+import WalletInputComponent from './WalletInput';
+import { getEllipsisTxt } from '../../web3utils';
+
+const { WalletInput } = composeStories(stories);
+
+let container: HTMLDivElement;
+const testValue = '';
+const testPlaceholder = '';
+const testInputId = 'test-input';
+const testLabelId = 'test-label';
+const testDivId = 'test-div';
+const testEllipsisInputId = 'test-ellipsis-input';
+const testBlockieId = 'test-blockie';
+const testEvent = jest.fn();
+
+test("should conditionally render 'empty / filled' className", () => {
```

Comment:
niiiiice you used render and screen, i assigned you `<Input />` React18 task next, sorry its beefy but you know Input well now =]

You can close this comment